### ISSUE TITLE
More Simple Language for Revenue vs. Expenses

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/buildabudget#6]

**Comment by graceatkins** [comment]
Potential definitions found online: 

-**The Revenue Budget** is an estimate
of how much money the city government
will take in during the year. This
includes collections of taxes such as the
general property tax and sales tax. In addition to taxes, the revenue budget includes money generated by the city departments for services such as
licenses and permits. 

-**The Expense Budget** funds current city
government services, including the salaries
of government employees, pensions, debt
service, and routine operating expenses
such as copier paper, pens and pencils,
printing services, rent and utilities. It is the Expense Budget that people
generally think of as “the city budget.”

**Comment by omnignorant** [comment]
I think it's worth starting with simple, though potentially, strategically imprecise statements for top-level clarity, then providing opportunities for a deeper dive as a click through or embedded into the experience.

So for example, what if top-level:

- The Revenue Budget is what Austin needs to collect;
- The Expense Budget is what Austin hops to spend;
- In a balanced budget, the expense budget equals the revenue budget;

Then, in the app, if you hover over, click an expansion button, or maybe in the next page of the app flow you can go a level deeper:

- Revenue Budget is a breakdown of the money Austin expects to collect in taxes, licenses, permits, fees, etc.  
- Expense Budget is a breakdown of the money Austin expects to spend on salaries, pensions, rents, pencils, paper, etc.

Then you can always add details and examples by doing things like color-coding revenue items and expense items, or add little quizzes (for points) that ask people whether they can separate revenue items and expense items in a list.

**Comment by omnignorant** [comment]
Some of these little quizzes could do things like ask people when they reduce funding for something, whether they'd support something else, like fees from private service providers, etc.

**Comment by krmc3182** [comment]
Hi,
So my impression is that the overall goal of this project is to get people thinking about the budget in terms of trade offs. I think, before they start the game, if you get people thinking about the trade offs they have to face in their own finances the game will be more effective. Here's what I came up with:

Most of us sit down and think about how much we can afford to spend on different things each month. With $1,000 of disposable monthly income we set aside a couple hundred for food a couple hundred for entertainment, etc, etc. The more we spend in one area the less we can spend in others. The city of Austin is not operating with an unlimited budget, so they have to finance government departments like the Public Safety and the Park Services departments much the same way we finance our personal lives.

**The Revenue Budget** is simply the government’s estimated income, the money they’ll take in from taxes, fines from things like parking tickets, charges for services like specialized license plates, and other odd places.

**The Expense Budget** is how much the government expects to spend after they sit down a decide how much to spend on each project. To do that they have to consider things like the salaries of government employees, pensions, debt service, and routine operating expenses, such as, copier paper and pens and pencils. This is what people are typically talking about when they talk about the “city budget”

If the revenue budget is the greater than the expense budget, that is to say if the government expects to make more money than expects to spend, then we can expect the budget to be in **surplus**. If the revenue budget is less than the expense budget we can expect the budget to be in **deficit**. If the two budgets are the same we can expect the budget to be **balanced**.
